redhat-developer/odo
# Fixing #1160: Fixing error message user creating application without project

## What is the purpose of this change? What does it change?

If user does not have an active project or is using project 'default',
then error message appropriately tells user to create or set a new
project.

'default' project is not recommended anyway as its fallback project for
any non-assigned project case and user may not have permissions for same

## Was the change discussed in an issue?
fixes #1160 

## How to test changes?
Login to openshift cluster without an active project or 'default' project set as active project and do

`odo create redhat-openjdk18-openshift:1.4` or any other command that attempts to create or delete anything other than a project

Expected output : 

```
# No current project
could not get current project.  Please create or set a project
    odo project create|set <project_name>
# 'default' is set as current project
current project is 'default'. Please create or set a different project
    odo project create|set <project_name>
```
Signed-off-by: Mohammed Zeeshan Ahmed <mohammed.zee1000@gmail.com>
